Makefile

```diff
@@ -94,6 +94,12 @@ $(SDK_BUILD_MARKER):
 	mkdir -p $(BUILD_DIR)
 	echo "$(BUILD_DATE)" > $@
 
+.PHONY: all-chips
+all-chips:
```

Comment:
Should this depend on SDK_BUILD_MARKER?

Reply:
I think it could.
I already committed without it.
If we feel that it could improve performance we can change it.